packages/web/components/cards/bond-card.tsx

```diff
@@ -110,7 +110,7 @@ export const BondCard: FunctionComponent<
                   .toString(),
               })}
             </span>
-            <span className="flex items-center gap-1 text-subtitle1 font-subtitle1 text-osmoverse-400">
+            <span className="subtitle1 flex items-center gap-1 font-subtitle1 text-osmoverse-400">
```

Comment:
Let's make sure both `text-` and `font-` are replaced for these classes. I also see `font-subtitle1` here. Should help reduce clutter